IDE/Renesas/cs+/Projects/t4_demo/t4_demo.mtpj

```diff
@@ -0,0 +1,1240 @@
+<CubeSuiteProject>

```

Comment:
I don't see this file in the .am file.

Reply:
Please respond to the comment about the demo project not listed in the .am file.

Reply:
Sorry to be very late.

Reply:
Thanks.